README.md

```diff
@@ -7,3 +7,291 @@ A highly robust Discord management bot with a featureset designed around content
+
+To register custom slash commands with the server you will need to interact with the discord API. In the examples folder there is an example script which will create a / command for creating a prediction.
+
+### Update Script
+- `Application ID`: You can get this ID from the General Information page when creating the application
+- `GUILD ID`: This is the same ID from the config.ini
+- `TOKEN`: The same token used in the config.ini
+
+## Run the bot
+Ensure both the MySQL and Redis instances are still running
+```bash
+podman ps
+```
+Make sure you have activated the python venv
+```bash
+[RoboBanana]$ source .venv/bin/activate
+```
+Start the hypercorn server
+```bash
+[RoboBanana]$ ./startServer.sh
+```
+You'll see output like:
+```bash
+2023-08-26 22:20:32 INFO     apscheduler.scheduler Adding job tentatively -- it will be properly scheduled when the scheduler starts
+2023-08-26 22:20:32 INFO     apscheduler.scheduler Added job "keep_alive" to job store "default"
+2023-08-26 22:20:32 INFO     apscheduler.scheduler Scheduler started
+2023-08-26 22:20:32 INFO     discord.client logging in using static token
+[2023-08-26 22:20:32 +0100] [46277] [INFO] Running on http://0.0.0.0:3000 (CTRL + C to quit)
+2023-08-26 22:20:32 INFO     hypercorn.error Running on http://0.0.0.0:3000 (CTRL + C to quit)
+2023-08-26 22:20:33 INFO     discord.gateway Shard ID None has connected to Gateway (Session ID: ba8b122009a7d9aaa2c387f51f43ab84).
+2023-08-26 22:20:35 INFO     server.util.discord_client Logged in as my_robobanana#5138 (ID: 1143616097560580116)
+
+```
+
+### Starting the actual bot
+#### Command line
+If you want to just run the bot and watch for logs without using breakpoints to step through. You can open a new terminal and simply run:
+```
+[RoboBanana]$ python bot.py
+```
+#### VSCode Debugging
+If you would like to use VSCode debugging and have copied the launch config file from previous steps or have your own then you can click the icon on the left that looks like a small bug on top of a play symbol.
+
+Finally click the play symbol at the top of the new pane with the words RoboBanana next to it.
+
+You'll see a terminal in VSCode open showing the bot activate and start listening for commands.
+```bash
+RoboBanana]$ source RoboBanana/.venv/bin/activate
+(.venv) [RoboBanana]$  /usr/bin/env RoboBanana/.venv/bin/python .vscode/extensions/ms-python.python-2023.14.0/pythonFiles/lib/python/debugpy/adapter/../../debugpy/launcher 48971 -- RoboBanana/bot.py 
+2023-08-26 22:31:09 WARNING  discord.client PyNaCl is not installed, voice will NOT be supported
+2023-08-26 22:31:09 INFO     discord.client logging in using static token
+2023-08-26 22:31:10 INFO     discord.gateway Shard ID None has connected to Gateway (Session ID: 6bc608b4de3f9ede5868f1818af8093b).
+2023-08-26 22:31:12 INFO     root Logged in as my_robobanana#5138 (ID: 1143616097560580116)
+2023-08-26 22:31:12 INFO     controllers.temprole_controller [TEMPROLE TASK] Running expire roles...
+```
```

Comment:
Again would remove this so as not to tie our instructions to a single IDE

Reply:
removed